src/components/HamburgerDropdown/styles.ts

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
You don't need this new lines, and you should have new lines at the end of the file.